jekyll/_cci2/benefits-of-circleci.adoc

```diff
@@ -0,0 +1,52 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+  - Server v2.x
+---
+= Benefits of CircleCI
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Benefits of choosing CircleCI for CI/CD
+:icons: font
+:toc: macro
+:toc-title:
+
+
+[#benefits-of-circleci]
+== Benefits of CircleCI
+
+Organizations choose CircleCI because jobs run fast and builds can be optimized for speed. CircleCI can be configured to run very complex pipelines efficiently with sophisticated xref:caching#[caching], xref:docker-layer-caching#[docker layer caching], and xref:optimizations#resource-class[resource classes] for running on faster machines.
```

Comment:
```suggestion
Organizations choose CircleCI because jobs run fast and builds can be optimized for speed. CircleCI can be configured to run very complex pipelines efficiently with sophisticated xref:caching#[caching], xref:docker-layer-caching#[Docker layer caching], and xref:optimizations#resource-class[resource classes] for running on faster machines.
```